contentcuration/contentcuration/tests/test_garbage_collect.py

```diff
@@ -62,7 +62,9 @@ def test_deletes_associated_files(self):
             contentnode_id=c.pk,
             file_on_disk=ContentFile("test"),
             checksum="aaa",
+            file_format_id=file_formats.JPG
```

Comment:
It's necessary to specify the `file_format_id`, it'll try and guess and use `'.jpg'`, which isn't an allowed file format, so we get a `foreign key constraint` error the `file_format_id` should actually be `'jpg'`

Reply:
When you say it'll try and guess, are you referring to this code in File.save?

            if not self.file_format:
                self.file_format_id = os.path.splitext(self.file_on_disk.name)[1]

If so, seems like we should update this code to strip the period, as if I'm reading this code correctly, it won't work for any case until we strip that period out. 

Also, if it's possible, we may want to see if the extension is in the LE Utils file_formats dictionary before assigning it.